src/views/Header.js

```diff
@@ -306,30 +326,26 @@ const styles = StyleSheet.create({
     elevation: 4,
   },
   appBar: {
-    flex: 1
+    flex: 1,
+    position: 'relative',
   },
-  item: {
+  header: {
     flexDirection: 'row',
-    alignItems: 'center',
   },
-  title: {
-    bottom: 0,
-    left: 40,
-    position: 'absolute',
-    right: 40,
-    top: 0,
+  item: {
+    justifyContent: 'center',
   },
+  title: Platform.OS === 'android'
+    ? {
+      flex: 1,
+      alignItems: 'flex-start',
+    }
+    : null,
   left: {
-    bottom: 0,
-    left: 0,
-    position: 'absolute',
-    top: 0,
+    alignItems: 'flex-start',
```

Comment:
<img width="170" alt="screen shot 2017-02-17 at 09 13 04" src="https://cloud.githubusercontent.com/assets/2464966/23057624/4cff89a6-f4f1-11e6-88fb-d28189b74ad0.png">

Left button can be smaller than the space allowed for it. That's why we use `flex-start` on the container (`red` background). `alignSelf` would make sense to be applied to back button itself (blue background).

For right container, we have exactly same situation thus we use `flex-end`.